common/djangoapps/third_party_auth/models.py

```diff
@@ -618,6 +618,13 @@ class SAMLProviderConfig(ProviderConfig):
             "This is helpful for testing/setup but should always be disabled before users start using this provider."
         ),
     )
+    country = models.CharField(
```

Comment:
Kind of surprised there are no backend changes to actually use this field in some way. 🤔

Reply:
I think you may want to check around here:
https://github.com/edx/edx-platform/blob/master/common/djangoapps/third_party_auth/saml.py#L261

Also not sure if we need/want a default here: https://github.com/edx/edx-platform/blob/9cff06d76275bd49374a9f50499d964b1545410d/common/djangoapps/third_party_auth/models.py#L694

Reply:
For general understanding, I believe field mapping is core to SAML. Like the act of mapping these fields is using the standard of SAML and that's how it just *works*, but you can also take a look at the underlying authentication mechanism: https://github.com/python-social-auth (specifically social-core, social-app-django), but I don't think that will totally help your understanding here or figure out a true testing mechanism.

Reply:
did we resolve this question?

Reply:
I don't think we want country to be a default here, but Talia does bring up a good point- `country` needs to be added to the list of attrs in the model's `get_config()`